Header header
float32 range_max   # Maximum range value [m]
int32 bins          # Number of bins of the returned echoes
float32 angle       # Angle of the current scan
uint8[] intensities # Intensities of the returned echoes